content/events/2022-11-01-developer-week.md

```diff
@@ -5,13 +5,65 @@ ends = "2022-11-18T17:00:00"
 timezone = "America/Los_Angeles"
 location = "Online Event"
 image = "https://storage.googleapis.com/media.urbit.org/site/events/developer-week.jpg"
-registration_url = ""
+registration_url = "https://www.meetup.com/urbit-sf/"
 description = ""
 
 +++
 
-It's been almost two months since our second Assembly, and the last cohort of Hoon School Live has just finished. 2022 is drawing to a close and the Urbit developer community is getting bigger every day. This calls for a celebration. 
+It's been almost two months since our second Assembly, and the last cohort of Hoon School Live has just finished. 2022 is drawing to a close and the Urbit developer community is getting bigger every day. This calls for a celebration.
```

Comment:
```suggestion
It's been almost two months since Assembly and the last cohort of Hoon School Live has just finished. The year is drawing to a close and the Urbit developer community is getting bigger every day. This calls for a celebration. 
```